### DISCUSSION TITLE
适配`https://member.bilibili.com/`

### DISCUSSION THREAD

**Discussion by TC999** [BewlyBewly/BewlyBewly#849] [Ideas]
有没有适配<https://member.bilibili.com/>的想法，一直用DarkReader，发黄页面快看吐了
试着在`manifest.ts`里添加了这个网址，然而没有任何变化
我过几天做个css样式试下

**Comment by hakadao** [comment]
這邊我只打算用 filter: invert(1) hue-rotate(180deg) 解決，因為這一部份適應比較麻煩，即時合併了做好的pr後續維護也是過大工程且收益很少

**Comment by hakadao** [comment]
已適配
